unattended-upgrade

```diff
@@ -986,19 +988,6 @@ def wrap(t, width=70, subsequent_indent=""):
     return out
 
 
-def setup_apt_listchanges(conf="/etc/apt/listchanges.conf"):
```

Comment:
Why is this dropped? Is apt-listchanges behaving differently these days? This was (a long time ago :) fixing a real bug, see debian 579733. In the old days APT_LISTCHANGES_FRONTEND=none was needed because if people had set their frontend to pager iirc apt-listchanges would block an unattended-upgrade run because it would present the changes in a pager waiting for someone to confirm them. Is this no longer true?

Reply:
The patch was offered in https://bugs.launchpad.net/ubuntu/+source/unattended-upgrades/+bug/1633098 , but since the bugs were about sending mails and not about blocking I did not check the potential regression with pager. For now I'm dropping this change from the PR and if pager works well, too, I'm scheduling it for a later release.